pkg/migrations/backfill/batcher.go

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package backfill
```

Comment:
it would be good to have some test for this package

Reply:
The backfill behaviour is already tested by all the operations that perform backfills on start (setting `NOT NULL`, change type...).

Unless you think it's worth adding some larger performance tests for how long it takes against large table, I'd rather test the user-visibile behaviours that use backfill, rather than adding tests for backfill in isolation.